tests/flow/test_prev_rdb_decode.py

```
@@ -1,12 +1,267 @@
+            f.write(dump)
+            f.flush()
+
+        # stop db
+        stop_db(container)
+
+        return dump
+    else:
+        with open(path, 'rb') as f:
+            return f.read()
+
 class test_prev_rdb_decode(FlowTestsBase):
```

Ruff

`FlowTestsBase` may be undefined, or defined from star imports

(F405)